packages/gatsby-theme-docs/src/components/card.js

```diff
@@ -23,6 +23,11 @@ const Container = styled.li`
   padding: ${designSystem.dimensions.spacings.m};
   border-radius: ${designSystem.tokens.borderRadiusForCard};
 `;
+const ImageContainer = styled.div`
+  svg {
+    width: 100%;
+  }
```

Comment:
```suggestion
  svg, img {
    width: 100%;
    height: auto;
  }
```

The SVGs have a  height and widht hardcoded, that's how it often is when you get them from design.  If you just set the width to 100% but leave the hardwired height of 80px  then the height wins.  That's not SVG or ViewBox specific, that's plain old CSS. 

Plus, I suggest to addresss <img> too because the image component could be a pixel image, too.  Or, just target any first child.